opentreemap/treemap/templates/treemap/map-add-tree.html

```diff
@@ -35,22 +35,26 @@ <h4 class="summaryAddress"></h4>
                 <hr/>
                 <label>{% trans "After I add this tree..." %}</label>
                 <div>
-                    <input type="radio" name="addFeatureOptions" value="copy" id="addtree-addsame" /><label for="addtree-addsame">
+                    <input type="radio" name="addFeatureOptions" value="copy" id="addtree-addsame"
+                           data-event-category="add-tree" data-event-action="done-add-copy" /><label for="addtree-addsame">
                         {% trans "Add another tree with these details" %}
                     </label>
                 </div>
                 <div>
-                    <input type="radio" name="addFeatureOptions" value="new" id="addtree-addnew" /><label for="addtree-addnew">
+                    <input type="radio" name="addFeatureOptions" value="new" id="addtree-addnew" 
+                           data-event-category="add-tree" data-event-action="done-add-new" /><label for="addtree-addnew">
                         {% trans "Add another tree with new details" %}
                     </label>
                 </div>
                 <div>
-                    <input type="radio" name="addFeatureOptions" value="edit" id="addtree-viewdetails" /><label for="addtree-viewdetails">
+                    <input type="radio" name="addFeatureOptions" value="edit" id="addtree-viewdetails"
+                           data-event-category="add-tree" data-event-action="done-edit" /><label for="addtree-viewdetails">
                         {% trans "Continue editing this tree" %}
                     </label>
                 </div>
                 <div>
-                    <input type="radio" name="addFeatureOptions" value="close" id="addtree-done" checked /><label for="addtree-done">
+                    <input type="radio" name="addFeatureOptions" value="close" id="addtree-done" checked
+                           data-event-category="add-tree" data-event-action="done" /><label for="addtree-done">
                         {% trans "I'm done!" %}
```

Comment:
These only add events if the user changes "After I add this tree". Seems like it would be better to have an event on the "Done" button.